_themes/rtd-blog/social.html

```diff
@@ -56,10 +56,13 @@ <h3> Follow us </h3>
 
 <ul>
 <li>
-<a href="http://blog.readthedocs.com/archive/atom.xml" rel="me">Atom Feed</a>
+<a href="http://blog.readthedocs.com/archive/atom.xml" rel="me">Atom Feed 📶️</a>
```

Comment:
```suggestion
<a href="http://blog.readthedocs.com/archive/atom.xml" rel="me">📶️ Atom Feed</a>
```

I would prefix all the texts with the emojis so they are aligned.

Reply:
I do usually prefer that but for some reason the other way looked better (to me)

![image](https://user-images.githubusercontent.com/374612/210830624-70670ef0-eb29-4296-b597-2f94abbd86c5.png)